config/initializers/fog.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  if !Rails.env.test? && !Rails.env.development?
+    config.storage = :fog
+    config.fog_credentials = {
+    :provider               => 'AWS',
+    :aws_access_key_id      => "#{Rails.application.secrets.aws_access_key_id}",
+    :aws_secret_access_key  => "#{Rails.application.secrets.aws_secret_access_key}",
+    :region                 => 'us-east-1',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.